crates/nargo/tests/test_data/6_array/proofs/p.proof

```diff
@@ -0,0 +1 @@
+2bbec2c66cbbaac2ef11a79bb4d452cb6df1dc8a7dd274fbc8b41a2af2279a442f8620bfe65f9725debc4d7ac5783998749309917073fd0ffe44c883a7e475100241d0b31082159beb689a11d295e508b779c8d012a473e996b2e69140acdfa613a62f16d9ce42ff0a91418bb9ad92d12e16cb8e9458a4d83017a0f887afb4c32e543e58ed2be6949f05b10d098610b56ddc9eabf6bdf474afd1d2ccfa52e6f21d5f74526b02ab7111f0a09241b6e0875e93e5657775237940cce02dcc245da42d0363fb60892e25b9dc82bdc0aa0f97477feb42c907935615cc9c53bb1b976901b86d7cf584503c856f1f3fc7f9dfa77a97865230666dc9d68a9bef5895e34a0650b8f938914236b28d8636724e83d0df9461f221c989ad067c0d3d871d24d016b0b8aac568074163eb12b2a9d5892623833633c1eb4afdec390e6eccf2cb6d2d5ea1097a2af39145ac88dcbb68dc34723514bb956ca75aefd88f61c4e424de0fdf146e9e2199572cc7cb20b908e1916d19207ce5ddfe3cf70af2bbecf826541e43d0b885ebe48824556a38153ca40ca48dccb925ab76466f05fa43aff5be391ce22ed792b0a8e061b417b957796164abd1ca93aba7f63095ee93b0f28549760387e00aeaee779d6c9ae74709d935ba91d3db2f13c3e89d7550952a25a6e6a72866523ead8433b9616bbd83b765b1b63bc5aba2085ddffee46ab5f6113b7d3d06ee880f6f3f98a64d04e419446099305fb105dbc18ddcfbb3991c5afa2c86642ae0cfcbc858342a960bfe39ef968486feaaf100ceea316cad90d44ea1b38943054dc931daaad326053dfead119e5025d032a471869e4e40ba55f5da3dea8fa026aa97a4c363a43f254e612041e956eadcdd30647908850acbb9336bb75f2e9418da4fe3a3ecb31211470d0ef0d6fc33c0039911c174cb137bbf005e685b5e4221af1513cc7d0c2c703b572c8ef9a7c91407abf5f6e56ca0937a0626b127040c19c4daa628bfcb0004b37675d37f12784a96a55404aa20c4534475cd13c9476512e042bfda07fad544f20a8db4c6499888cae01cbafceda5099eefe8e3e0a75221166a10203aeae8d6fa17f3bec644129d19d1faf7cde8b1aa3e32d8adfcf5b71110121fb880bc5dddda2fb093c3964d6aff6209dcc60bb04b504f4f57f6fa9429dae30435ba92eed01a5a6d36b4cddb858e30d28f15603aa6806b66f1ac4ea629ada86016980bf62b93590cafc09f1cb5bbc289d54f7f8062e6ca0db4ab45e91a377343ea9febd396c103f1e254a63dfea004634842380c164c1cf7fb5b05fb0305166e76b64a75d786f6ac9771a5ff58cf0bff51f5cf91ae1fcaf1cf26eefd2a764c3c7c27577dbc5604e88dbc4e1ec81f78942e619c71e731351c605caea405a55d4d50629b2940eee12c94cf50c35eb8da2f91adca631ddacef43f3d03c70c86d05a9c598e318e15397397f5ff3ce61a3f1651804fe9876ba07d2328f981186aa71921c48c5a68f30647df6bfcc1901972379c3bd2ce74903196fe3581bf0bbae42140a605f1b3f75bb84bafb6528859e0a00e088a033a117f73ba1cc90b174966f854cc0dc5c0e2a0e3fb9395f8f4c57c86d588d28802ab2b572efe45a226cc5b500f431cc8dccd6e664e1f8c443cba137e213af3c0c993c9c51d728cbe2250132e08617abac026e0ad902215a7e0e10be81fa7deeb7989b3f43fecbc8d
```

Comment:
Looks like this was committed by accident.

Reply:
Indeed